### PR TITLE
add documentation from example to codebase

### PR DESCRIPTION
**Pull request by thijstriemstra**
you added such lovely documentation to the example, but the actual code has no documentation at all. I think maintaining the documentation in many examples, and duplicating it, makes it hard to maintain so I would suggest to shorten the docs in the examples and properly maintain docs in codebase.

My editor also removed trailing whitespace, would be good if yours does the same.

### CODE REVIEW

File: src/ADS1115_WE.h [hunk 1]
```diff
@@ -102,28 +102,113 @@ typedef enum ADS1115_STATUS_OR_START{
...
+    */
 	void setCompareChannels(ADS1115_MUX mux);
+
 	bool isBusy();
 	void startSingleMeasurement();
 	float getResult_V();
 	float getResult_mV();
+
+	/* With this function the alert pin will be active, when a conversion is ready.
+     * In order to deactivate, use the setAlertLimit_V function
```
Comment (thijstriemstra) [inline src/ADS1115_WE.h:206]: @wollewald this docstring mentions the `setAlertLimit_V` but i can't find it. what should the correct name be?

**Comment by thijstriemstra** [comment]
I also added a travis configuration file so the library gets build on a remote server and lets us know when there's something wrong with the build. You need to enable travis for this repository though (i dont have permissions), see https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github

File: src/ADS1115_WE.h (hunk 1 above)
Comment (wollewald) [reply src/ADS1115_WE.h:206]: This is simply a left over from first versions, which were not published. The setAlertLimit_V function does not exist (anymore). 
Internally the conversion ready alert is activated when the MSB of the upper alert limit is set to 1 (which means a negative value) and the MSB of the lower alert limit is set to "0" (which means a positive value). A bit strange, but this is how the ADS1115 works.
Therefore there are two ways to disable the conversion ready alert:
1) if you chose a combination of upper and lower limit which makes sense (upper limit higher than lower limit), using setAlertModeAndLimit_V --> then you use the alert pin for limit alert
2) setAlertPinMode(ADS1115_DISABLE_ALERT); --> then you don't use the alert pin at all
I am not sure if I should write this in detail into the comment. I think I will only mention option 2. What do you think?

**Comment by thijstriemstra** [comment]
@wollewald can you enable travis for this repository? see https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github

**Pull request merged**